thinkst/canarytokens
# Use token icon color palette for the celebration confetti

## Proposed changes
- Add custom confetti colours for each token type using the icon's palette

## How to test
- Pull the branch
- Create different tokens and confirm the confetti matches the token icon colours

## Demo

https://github.com/thinkst/canarytokens/assets/145110595/9602ba0f-a8ac-4137-ab03-fcf4e3077c26
